### PR TITLE
Improve developer workflow

### PR DESCRIPTION
**Pull request by lrascao**
Just some niceties i found missing banging my head around while standing up a node

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -421,6 +436,9 @@ internal-build: VERSION REVISION internal-compile-deps endpoints
 internal-start:
 	@./_build/$(KIND)/$(CORE) start
 
+internal-console:
+	@./_build/$(KIND)/$(CORE) console
```
Comment (lrascao) [inline Makefile:440]: maybe more people prefer running interactive console better

File: config/dev1/sys.config [hunk 2]
```diff
@@ -36,8 +36,8 @@
   {aecore, [
       {sync_port, 3015},
 
-      {peers, [<<"aenode://pp_23YdvfRPQ1b1AMWmkKZUGk2cQLqygQp55FzDWZSEUicPjhxtp5@localhost:3025">>,
```
Comment (lrascao) [inline config/dev1/sys.config:39]: these hardcoded addresses did not match the generated-on-startup peer keys, so nodes weren't able to connect

File: rebar.config [hunk 3]
```diff
@@ -200,7 +200,46 @@
...
+                            {vm_args, "./config/dev2/vm.args"},
+                            {overlay, [{mkdir, "{{output_dir}}/data"},
+                                       {copy,
+                                        "_build/dev2/bin/check_config",
+                                        "{{output_dir}}/bin/check_config"},
+                                       {copy,
+                                       "_build/dev2/lib/aeutils/priv/aeternity_config_schema.json",
+                                       "{{output_dir}}/data/aeternity_config_schema.json"},
+                                       {copy,
+                                       "config/dev2/keys/{peer_key,peer_key.pub}",
```
Comment (lrascao) [inline rebar.config:223]: relx supports this copy format `{X,Y,Z}`, might be worth using in other copy sections above and reduce the verbosity

File: config/dev1/sys.config (hunk 2 above)
Comment (lrascao) [reply config/dev1/sys.config:39]: i've noticed now that these are set at `aecore_suite_utils` so they shouldn't be changed, it's probably the `peer_key` and `peer_key.pub` that need to contain these keypairs

**Comment by lrascao** [comment]
dev{1-3} keypairs extracted from `aecore_suite_utils` and encrypted to disk with `peer_password`

**Review by cytadela8 — APPROVED** [review]
Works and I like it :smiley:

**Pull request closed without merging**